google/resource_container_cluster.go

```diff
@@ -747,6 +774,11 @@ func resourceContainerClusterRead(d *schema.ResourceData, meta interface{}) erro
 			"cluster_ca_certificate": cluster.MasterAuth.ClusterCaCertificate,
 		},
 	}
+	if len(cluster.MasterAuth.ClientCertificate) == 0 {
+		masterAuth[0]["client_certificate_config"] = []map[string]interface{}{
+			{"issue_client_certificate": false},
+		}
```

Comment:
If you set `issue_client_certificate` to `true` in a else statement to this if,  I think it would simplify `masterAuthClientCertCfgSuppress` since it will only need to handle one case:
`[{ "issue_client_certificate": true}] --> []`

Reply:
I wanted to do it like that first :/ The problem with that approach is that we don't actually have access to the old data (i.e. issue_client_certificate=true) in the diff suppress function, and because the new value is nil, it doesn't attempt to do the diff of issue_client_certificate.

Reply:
I see 👍